sql/bgw_scheduler.sql

```diff
@@ -73,6 +73,16 @@ AS '@MODULE_PATHNAME@', 'ts_policy_compression_proc'
 LANGUAGE C;
 
 /* continuous aggregates policy */
+CREATE OR REPLACE FUNCTION add_refresh_continuous_aggregate_policy(continuous_aggregate REGCLASS, start_interval "any",  end_interval "any" , schedule_interval INTERVAL, if_not_exists BOOL = false)
```

Comment:
I am wondering if we shouldn't use a "default" `start_interval` (similar to current `refresh_lag`), which is some factor of bucket size. 

Also not sure about the parameter names, but we can discuss that separately.

Reply:
we save bucket_width in the catalog. so we could add a default here.

Reply:
This reminds me of something related: do we support the `time_bucket` offset for caggs? In that case, where do we store the offset?